jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -30,6 +30,7 @@
 import com.github.victools.jsonschema.module.javax.validation.JavaxValidationOption;
 import com.github.victools.jsonschema.module.swagger15.SwaggerModule;
 import com.github.victools.jsonschema.module.swagger15.SwaggerOption;
+
```

Comment:
We seem to have different IDE settings in regards to auto-managed imports 😉

Reply:
If you can share the required lay-out, I can configure it in my IntelliJ.
I leave it for you to correct the file.